regtests/0192_client_headers/client_headers.adb

```diff
@@ -1,7 +1,7 @@
 ------------------------------------------------------------------------------
 --                              Ada Web Server                              --
 --                                                                          --
---                     Copyright (C) 2008-2017, AdaCore                     --
+--                     Copyright (C) 2008-2018, AdaCore                     --
```

Comment:
2018 ?

Reply:
It's a cherry pick (as-is) of another commit from 2018.